src/cli/organize.py

```
@@ -16,6 +17,87 @@
+    lives inside this function so ``fo version`` etc. don't pay the
+    cost (#404 fast-path). On any failure (import, broken YAML,
+    missing creds) we fall through to ``"ollama"``, the safe default
+    that prevents the Ollama-flood case #408 was originally about.
+    """
+    try:
+        from config.provider_env import get_model_configs
+    except ImportError:
+        return "ollama"
+    try:
+        text_cfg, _ = get_model_configs()
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)